pkg/apis/policy/validation/validation.go

```diff
@@ -33,8 +33,15 @@ import (
 	core "k8s.io/kubernetes/pkg/apis/core"
 	apivalidation "k8s.io/kubernetes/pkg/apis/core/validation"
 	"k8s.io/kubernetes/pkg/apis/policy"
-	"k8s.io/kubernetes/pkg/security/podsecuritypolicy/seccomp"
-	psputil "k8s.io/kubernetes/pkg/security/podsecuritypolicy/util"
+)
+
+const (
+	// AllowAny is the wildcard used to allow any profile.
+	seccompAllowAny = "*"
+	// DefaultProfileAnnotationKey specifies the default seccomp profile.
+	seccompDefaultProfileAnnotationKey = "seccomp.security.alpha.kubernetes.io/defaultProfileName"
+	// AllowedProfilesAnnotationKey specifies the allowed seccomp profiles.
+	seccompAllowedProfilesAnnotationKey = "seccomp.security.alpha.kubernetes.io/allowedProfileNames"
```

Comment:
These are being removed too, right? As part of https://github.com/kubernetes/kubernetes/pull/109819

Reply:
This API code is staying unchanged for 1.25 and will be removed in 1.26. The other PR is on my radar to review as well